ROADMAP.md

```diff
@@ -1,40 +1,64 @@
 # Roadmap
 
 This document defines a high level roadmap for Rook development.
-The dates below are subject to change but should give a general idea of what we are planning.
+Any dates listed below are subject to change but should give a general idea of what we are planning.
 We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 0.8
-
-* Multiple storage backends
-  * Full design
-  * Refactor code base and repositories to enable
-  * Consider support for Minio, potentially early support for other backends time permitting
-* Run on arbitrary PVs
-* Remove Rook API and CLI
-* Run with Least Privileged design
-  * Improved support for OpenShift
-* Support Kubernetes 1.7+ only
-* Ceph features and improvements
-  * Disk management (1 OSD per pod, adding/removing disks)
-  * Placement group balancer support (ceph-mgr module)
-  * Mon reliability (one mon per node, failing over too fast, etc.)
-
 ## Rook 0.9
 
-* Durability of state (local storage support, config can be regenerated)
-* Custom resource validation, progress, status
-* Design for Volume Snapshotting and policies (consider aligning with SIG-storage)
-* Run without privileged containers
-* Ceph features and improvements
-  * Mimic support
-  * Improved data placement and pool configuration (CRUSH maps)
-  * Dynamic Volume Provisioning for CephFS
-  * ceph-volume provisioning for OSDs
-  * Ceph CSI plug-in that runs everywhere (rook-ceph-agent uses ceph-fuse, nbd-rbd / tcmu runner)
-* Cluster and Pool types are declared Beta
+* Update project governance policies [#1445](https://github.com/rook/rook/issues/1445)
+* Add Core Infrastructure Initiative (CII) Best Practices [#1440](https://github.com/rook/rook/issues/1440)
+* Integrate a more robust controller framework (e.g., CoreOS Operator SDK or Kubebuilder) [#1981](https://github.com/rook/rook/issues/1981)
+* Build and integration testing improvements
+  * Increase PR quality gates (e.g., [vendoring verification](https://github.com/rook/rook/issues/1822), [license scanning](https://github.com/rook/rook/issues/1993), etc.)
+  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
+  * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
+  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
+  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
+* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
+* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
+* Support for Kubernetes 1.9+
```

Comment:
@travisn I saw this item in your notes, but I'm not sure what the specific work items or issues here are. Can you elaborate?